examples/chainer/Dockerfile.example.chainer

```diff
@@ -0,0 +1,13 @@
+FROM pai.build.base:hadoop2.7.2-cuda8.0-cudnn6-devel-ubuntu16.04
+
+#set LC_ALL
+RUN export LC_ALL=C
```

Comment:
`RUN export` in Dockerfile  won't persist across images, pls use `ENV` instead. You can refer to https://stackoverflow.com/questions/33379393/docker-env-vs-run-export.

Reply:
Thanks, Yifan! Done!